webui/react/src/components/kit/Select.tsx

```diff
@@ -0,0 +1,163 @@
+import { Select as AntdSelect } from 'antd';
+import type { DefaultOptionType, LabeledValue, RefSelectProps, SelectValue } from 'antd/es/select';
+import React, { forwardRef, useCallback, useMemo, useState } from 'react';
+
+import Label, { LabelTypes } from 'components/Label';
+import Icon from 'shared/components/Icon/Icon';
+
+import css from './Select.module.scss';
+
+const { OptGroup, Option } = AntdSelect;
+
+export { Option, SelectValue };
+
+type Options = DefaultOptionType | DefaultOptionType[];
+export interface Props<T = SelectValue> {
+  allowClear?: boolean;
+  defaultValue?: T;
+  disableTags?: boolean;
+  disabled?: boolean;
+  enableSearchFilter?: boolean;
+  filterOption?: boolean | ((inputValue: string, option: LabeledValue | undefined) => boolean);
+  filterSort?: (a: LabeledValue, b: LabeledValue) => 1 | -1;
+  id?: string;
+  label?: string;
+  maxTagCount?: -1 | 0 | 'responsive';
```

Comment:
These variations aren't in the design kit. You removed the only variation that used it as "redundant" but if it's actually redundant then the code should be removed as well. If it's not redundant then the variation should show how it's different.

I also see quite a few other parameters that aren't reflected in the UI Kit (placeholder, enableSearchFilter, placement, etc).

Reply:
All prop variations, that affect the design as well as clearer examples have been added.